.ci/pipeline/ci.yml

```diff
@@ -42,6 +42,9 @@ variables:
   PYTHON: 'python'
   ARGS: '1'
   SHELLOPTS: 'errexit:pipefail'
+  ISORT_VERSION: 5.13.2
```

Comment:
Could this file somehow take the versions from the precommit .yaml? Would be nicer to have the versions in only one place instead of having a CI check that they match.